default/scripting/buildings/STARLANE_BORE.focs.txt

```diff
@@ -5,27 +5,38 @@ BuildingType
     buildtime = 1 + Target.System.NumStarlanes
     location = Planet
     effectsgroups = [
+        [[SPECIES_LIKES_OR_DISLIKES_BUILDING_STABILITY_EFFECTS]]
```

Comment:
Note sure whether having (dis-)likes for temporary buildings makes much sense, but Bore and Nexus should either both support them or both do not.

Reply:
probably not necessary